src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = {
+  goods: string[];
+};
+
+const GoodsList: React.FC<Props> = ({ goods }) => {
+  return (
+    <ul className="goodList">
+      {goods.map(good => (<li data-cy="Good">{good}</li>))}
```

Comment:
Add a unique 'key' prop to each 'li' element to avoid React warnings and improve performance.